<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>221</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Magnetic Field Control Window</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources/nSOTScannerResources.qrc">
    <normaloff>:/nSOTScanner/Pictures/jj_schema.svg</normaloff>:/nSOTScanner/Pictures/jj_schema.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">#centralwidget{
background-color: rgb(0, 0, 0);
}


QLabel{
color:rgb(168,168,168)
}

QLineEdit{
border: 1px solid rgb(100,100,100);
background-color:rgb(230,230,230);
}

QTextEdit{
border: 1px solid black;
background-color: black;
color:rgb(168,168,168);
}

QRadioButton{
color:rgb(168,168,168)
}</string>
   </property>
   <widget class="QPushButton" name="push_Servers">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#push_Servers{
background: rgb(144, 140, 9);
 border-radius: 4px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_field_static">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>82</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Field (T):</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>500</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line{
color:rgb(168,168,168)
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_setpoint">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>85</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ramprate">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>115</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="push_persistSwitch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Switch only works if the charging and persistant fields are the same. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">#push_persistSwitch{
background: rgb(0, 170, 0);
 border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_switchStatus">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Persistant</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_GotoSet">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>170</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#push_GotoSet{
color: rgb(168,168,168);
background-color:rgb(0,0,0);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}

QPushButton:pressed#push_GotoSet{
color: rgb(168,168,168);
background-color:rgb(95,107,166);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>GOTO
SET</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_GotoZero">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#push_GotoZero{
color: rgb(168,168,168);
background-color:rgb(0,0,0);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}

QPushButton:pressed#push_GotoZero{
color: rgb(168,168,168);
background-color:rgb(95,107,166);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>GOTO
ZERO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_hold">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#push_hold{
color: rgb(168,168,168);
background-color:rgb(0,0,0);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}

QPushButton:pressed#push_hold{
color: rgb(168,168,168);
background-color:rgb(95,107,166);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>HOLD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_clamp">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#push_clamp{
color: rgb(168,168,168);
background-color:rgb(0,0,0);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}

QPushButton:pressed#push_clamp{
color: rgb(168,168,168);
background-color:rgb(95,107,166);
border: 1px solid rgb(168,168,168);
border-radius: 5px
}</string>
    </property>
    <property name="text">
     <string>CLAMP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fieldval">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_current_static">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Current (A):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currentval">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_output_voltage_static">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>161</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Output Voltage (V):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_outputvoltage">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_setpoint_static">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Setpoint (T):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_ramprate_static">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Ramp rate (T/min):</string>
    </property>
   </widget>
   <zorder>push_Servers</zorder>
   <zorder>label_field_static</zorder>
   <zorder>line</zorder>
   <zorder>lineEdit_setpoint</zorder>
   <zorder>lineEdit_ramprate</zorder>
   <zorder>push_persistSwitch</zorder>
   <zorder>label_switchStatus</zorder>
   <zorder>push_GotoSet</zorder>
   <zorder>push_GotoZero</zorder>
   <zorder>push_clamp</zorder>
   <zorder>label_fieldval</zorder>
   <zorder>label_current_static</zorder>
   <zorder>label_currentval</zorder>
   <zorder>label_output_voltage_static</zorder>
   <zorder>label_outputvoltage</zorder>
   <zorder>label_setpoint_static</zorder>
   <zorder>label_ramprate_static</zorder>
   <zorder>push_hold</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../Resources/nSOTScannerResources.qrc"/>
 </resources>
 <connections/>
</ui>
